### DISCUSSION TITLE
Change Controls for the Platformer Template

### DISCUSSION THREAD

**Discussion by tajhans** [4ian/GDevelop#2673] [Enhancement ideas (and their technical discussions)]
When you make a new project and choose the platformer template some basic controls are used for the character. There is a d-pad and a jump button. However the d-pad has a up and down button so I see no use for the jump button. This also may confuse the user.

**Comment by Midhil457** [comment]
I think the up in the dpad makes the player climb the ladder. And the jump button makes the player jump

**Comment by arthuro555** [comment]
Hello, this place is meant for technical changes discussions. For regular feature requests, you are welcome to use [the forum](https://forum.gdevelop-app.com/c/gdevelop-general/feature-requests).
